website/content/api-docs/secret/pki.mdx

```diff
@@ -1602,3 +1602,32 @@ $ curl \
     --data @payload.json \
     http://127.0.0.1:8200/v1/pki/tidy
 ```
+
+# Cluster Scalability
+
+Most non-introspection operations in the PKI secrets engine require a write to 
+storage, and so are forwarded to the cluster's active node for execution.  
+This table outlines which operations can be executed on performance standbys 
+and thus scale horizontally.
+
+| Path                        | Operations          |
+| --------------------------- | ------------------- | 
+| ca[/pem]                    | Read                | 
+| cert/<em>serial-number</em> | Read                | 
+| cert/ca_chain               | Read                |
+| config/crl                  | Read                |
+| certs                       | List                | 
+| ca_chain                    | Read                |
+| crl[/pem]                   | Read                |
+| crl/pem                     | Read                | 
+| issue                       | Update <sup>*</sup> |
+| revoked/*                   | Read, List          |
+| sign                        | Update <sup>*</sup> |
+| sign-verbatim               | Update <sup>*</sup> |
+
+\* Only if the corresponding role has `no_store` set to true and `generate_lease`
+set to false.  If `generate_lease` and `no_store` are true, the lease is forwarded
```

Comment:
> If `generate_lease` and `no_store` are true

I don't think that's quite right.  How about "If `generate_lease` is true the lease creation will be forwarded to the active node; if `no_store` is false the entire request will be forwarded to the active node"?

Reply:
Sounds good.